pilot/pkg/features/pilot.go

```diff
@@ -246,14 +246,14 @@ var (
 
 	EnableAnalysis = env.RegisterBoolVar(
 		"PILOT_ENABLE_ANALYSIS",
-		false,
+		true,
```

Comment:
I don't think this is the right way to enable this for testing; rather, we should update https://github.com/istio/istio/blob/f483be4708f40f8f913b8e9fec1aaa11ca8f9306/tests/integration/iop-integration-test-defaults.yaml#L59.

Reply:
also probably merge test/integration/pilot/analysis into pilot/. Can be a followup